### DISCUSSION TITLE
Page is prefetched even with prefetch={false} on the Link

### DISCUSSION THREAD

**Discussion by knapcp3** [vercel/next.js#19566] [Help] [answered]
Hey, I have a next js app with two pages like:
`/pages/brand1/[[...subpages]].tsx`
`/pages/brand2/[[...subpages]].tsx`

I'm using Incremental Static Regeneration.
I want to disable prefetching of the page data.
I have the following link:
```
<Link
    href={`/brand1/[subpages]?subpages=products`}
    as={`/products`} 
    prefetch={false}
    shallow
 >
   <a>link</a>
 </Link>
```

When link appears in viewport there's no are no prefetching requests, but when I hover by mouse on the link prefetch requests are made, I don't know why and I don't want it.

Someone has any idea what I did wrong and how to prevent those prefetch requests on Link hover?

**Comment by michrome** [comment, marked as answer]
Next.js is supposed to prefetch the next page, even when prefetching is disabled, when a `<Link>` is **hovered**. The automated prefetch disabling is only supposed to affect the in-viewport prefetch.

See https://github.com/vercel/next.js/issues/18856

**Comment by elgharabawy** [comment]
Is there a way to stop prefetching completely including the one that happens on mouse enter? We have a use case where a product listing page pulls products from a third party service. Within one week, we racked up about 200K requests from their UAT domain only.
